src/DotVVM.Framework/ResourceManagement/ScriptResource.cs

```diff
@@ -13,7 +13,7 @@ namespace DotVVM.Framework.ResourceManagement
     public class ScriptResource : LinkResourceBase, IPreloadResource, IDeferableResource
     {
         public bool Defer { get; }
```

Comment:
Other properties of resources have public setter (and those setters are heavily utilized). Is there a reason why we don't have it here?

Reply:
We set `RenderPosition` to Body or Anywhere based on defer's value. This would just break if we added setter to `Defer`. Generally, setters only cause problems and we keep them for legacy reasons.